lib/coffee_place/cli.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require_relative '../coffee_place'
+require_relative './location'
+require_relative './cli_parser'
+require_relative './importer'
+require_relative './search'
+
+module CoffeePlace
+  # Parses and runs CLI application using supplied args.
```

Comment:
Why did you feel the need to explain what the classes do through additional comments?

Reply:
Rubocop has [a style cop](https://docs.rubocop.org/rubocop/cops_style.html#styledocumentation) that emits warnings when you haven't added a class comment description.
While disabling this cop is valid, when method names are explicit, I prefer using a consistent style.

Basically, some classes need no further explanation - their name is sufficient to understand their purpose.
Other classes need additional explanation as to their responsibility.
This would result in some classes having a documentation comment at the top, while other would
not have it - a minor style inconsistency.

This rubocop linter forces you to consistently add documentation at the top of any class,
making sure the style remains consistent.

I'm pretty sure the rubocop config for rails disables this, as convention over configuration makes
alot of class descriptions redundant.

tldr; Rubocop is being naggy as always! :stuck_out_tongue:

Reply:
I apologise for writing you a short novel in response to your comment ... I don't know how to stop myself from rambling sometimes :stuck_out_tongue:

Reply:
I usually disable that cop, I find it naggy too, but recently the thought occured to me that it can be much more useful in more complex projects, as it would help new developers understand the project implementation-wise more easily.

Out of curiosity - have you used this documentation cop in production? If so, has it helped the team understand the project better?

Reply:
We usually used the rails-rubocop config, which I think has this cop turned off.

What helped us understand new projects best were design documents and mockups,
which are initially created when first interacting with the client. These explain what the
project aims to be, which is usually more than its current implementation state.

Comments at the top of more complicated methods, or specs in which these methods are
tested usually help clarify specific parts of the code - but its hard to build a 'big picture' only 
from these.